File: Nian_Karl/QL_Nian_Karl/src/org/uva/sea/ql/AST/value/StringValue.java
package org.uva.sea.ql.AST.value;

public class StringValue extends AbstractValue<String> {

	private final String stringValue;

	public StringValue(String stringValue) {
		this.stringValue = stringValue;
	}
	
	@Override
	public String getValue() {
		return stringValue;
	}
}

File: CarlaErik/UvA.SoftCon.Questionnaire/AST/Types/StringValue.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace UvA.SoftCon.Questionnaire.AST.Types
{
    public class StringValue : Value<string>
    {
        public StringValue(string value)
            : base(value) { }
    }
}
